client/components/recipes.js

```diff
@@ -35,21 +40,16 @@ class Recipes extends React.Component {
     return (
       <div className="container">
         <h5 className="component-title">Recipes</h5>
-        <div>
+        <strong>Trending Keywords:</strong> <a href="#">Ramen</a>, <a href="#">Ramen</a>, <a href="#">Ramen</a>, <a href="#">Ramen</a>
+
+        <InstantSearch
+          appId="KUPHP9V5MI"
+          apiKey="8e465f8475198ae5cb2d621323e06fb4"
```

Comment:
Can we keep these keys in private files on our own machines instead of committing them to the repo?  Seems like that would be a more secure option.